### ISSUE TITLE
Your files are too powerful! [500MB Limit]

### ISSUE THREAD

**Issue by kazuviking2** [riolubruh/SplitLargeFiles#1]
When i try to send any file it just says something went wrong, If a file is smaller than 50mb it wont create chunks. I have YABD4Nitro enabled and when trying to send a 500mb or larger file it just says file is too big.

**Comment by riolubruh** [comment]
It should be fixed now in 41753d3a11773c89ff87f66f03283658d24027a7

**Comment by riolubruh** [comment]
oh wait nvm, it's only partially fixed

**Comment by riolubruh** [comment]
Fixed the first problem (If a file is smaller than 50mb it wont create chunks) with 109f53a.
The 500MB limit also seems to affect the original version of the plugin.

**Comment by riolubruh** [comment]
I actually managed to upload a 696MB file on a dev version, however, I think it's way too unstable to release.
![image](https://user-images.githubusercontent.com/54255074/215218916-f6721ba9-faab-4c12-af13-565248564023.png)
The upload process went through surprisingly well, however, it will seemingly only worked basically immediately after restarting Discord.
Then, after the upload process, the plugin was unable to hide all of the part file messages, so there ended up being a lot of this (below) afterwards.
![image](https://user-images.githubusercontent.com/54255074/215219399-85d8d7fa-bc11-4836-9eca-a7547d6c4f75.png)
Then finally, unsurprisingly, the file failed to reassemble automatically, however, to my complete surprise, all 87 of the DLFC files were successfully downloaded and by using the [online DLFC reconstructor](https://imthesquid.github.io/df-converter), I was able to confirm that _somehow_ the DLFC files were (miraculously) all correct.
